config/production/jenkins.patch

```diff
@@ -866,36 +866,8 @@ index 4be7c41..d0ed597 100644
      <artifactDaysToKeep>-1</artifactDaysToKeep>
      <artifactNumToKeep>-1</artifactNumToKeep>
    </logRotator>
-diff --git a/jenkins-master/jobs/project_functional/config.xml b/jenkins-master/jobs/project_functional/config.xml
```

Comment:
We removed project branch jobs a while ago. Instead of updating those jobs which are not needed anymore, I thought that I kill them in one go.